utilities/manual/usermanual_en.html

```diff
@@ -548,14 +548,13 @@ <h1 id="SECTION2">3. Compiling a document</h1>
 <h2 id="SECTION22">3.1 Compiling</h2>
 <p>The easiest way to compile a document is to use the "Compile" command or the "Build&amp;View" command ("Compile" button - shortcut : F6). You can select the default command via the "Configure TeXstudio" dialog.<br>(You can also launch each command one by one in the "Tools" menu).<br> Note : the "Clean" command in the "Tools menu" allows you to erase the files (dvi, toc, aux...) generated by a LaTeX compilation (except the ps and pdf files).</p>
 <p><IMG src="compile_toolbar.png" border="1" alt="compile_toolbar">
-<p><b>Warning: all your files must have an extension and you can't compile an "untitled" file or a file with a space in its name</b>.</p>
+<p><b>Warning: all your files must have an extension and you can't compile a file with a space in its name</b>.</p>
 <h2 id="SECTION23">3.2 The log files</h2>
-<p>With the "Quick Build" command, the log file is automatically displayed in the "Messages / Log file" pannel. While clicking on a number in the "Line" column, the cursor is placed on the corresponding line in the editor and the error is displayed. <br>
-Remark : a summary of the latex errors and warnings is displayed before the full log file.</p>
-<p><IMG src="doc15.png" border="1" alt="doc15"></p>
-<p>The "Next Latex Error"and "Previous LaTeX Error" commands allow to get to the errors detected during compilation.</p>
-<p>Lines with errors, warnings, bad boxes will be highlighted with red, yellow or blue background and you can jump between them using Ctrl+Up/Down. (Ctrl+Shift for errors only, Ctrl+Alt for warnings only, Alt+Shift for bad boxes only)<br>
-A tool tip will show more details of the mistake if you jump to a line (it is also shown if you move the mouse over the mark left from the line numbers).</p>
+<p>The Log tab helps with error handling. It provides the tabs Log File and Issues as well as some controls (see figure). The Issues tab displays a table of messages taken from the log file displayed below the table.</p>
+<p><IMG src="errorlog.png" border="1" alt="Display of Error Log"></p>
+<p>The first three controls can be used to choose whether errors (red), warnings (yellow) or bad boxes (blue) are shown, another whether to show error markers in the editor, and the last two to toggle back and forth between errors to be jumped. The shortcuts for this are Ctrl+Shift+Up/Down, accordingly Ctrl+Alt+Up/Down and Alt+Shift+Up/Down are used for warnings and bad boxes respectively. You jump unspecifically between the messages with Ctrl+Up/Down.</p>
```

Comment:
I would not number the buttons, especially as all are provided with tooltips.
Use the tooltips name (ideally with icon images, but that quickly get tedious...)

Reply:
The usage of the dialog is quite clear and the description just gives the user some basic understanding and orientation. And there is other work waiting. I'll add the icon tool tips...